bot/utils/dma_api.py

```diff
@@ -47,31 +47,53 @@ async def wrapped(search):
     return wrapper
 
 
-@get_api()
-async def get_article(number, articles):
-    return [
-        item
-        for item in articles
-        if item["series"] == str(number)[0]
-        and item["category"] == str(number)[1]
-        and item["digit"] == str(number)[2]
-    ]
-
-
-@get_api()
-async def get_series(series, articles):
-    return [item for item in articles if item["series"] == str(series)]
+async def get_search_api(query_string: str = None, series_number: int = None):
+    query = f'{"?query={" + re.sub(DMA_QUERY_REGEX, "", query_string) + "}" if query_string is not None else ""}'
```

Comment:
Any reason it's wrapped in a format string?
Couldn't something like this work?
```suggestion
    query = "?query={" + re.sub(DMA_QUERY_REGEX, "", query_string) + "}" if query_string is not None else ""
```